.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+Notes:
+* Pull requests will not be accepted until:
+-  The submitter has [accepted the online agreement here with a click through](https://developer.mbed.org/contributor_agreement/) 
+   or those that do not wish to create an mbed account, a slightly different agreement can be found [here](https://www.mbed.com/en/about-mbed/contributor-license-agreements/)
```

Comment:
I suggest slightly different language:
'...or for companies or those that do not wish to create an mbed account, a slightly different agreement can be found [here]'